services/app-api/src/postgres/postgresStore.ts

```diff
@@ -12,6 +12,8 @@ import type {
     ContractType,
     UnlockedContractType,
     CMSUsersUnionType,
+    RateQuestion,
```

Comment:
I think this type should have the suffix "Type". "RateQuestion" will be the graphql type